File: InPlan.Web/Modules/Parametros/Tecnicos/TecnicosForm.cs
using Serenity.ComponentModel;

namespace InPlan.Parametros.Forms;

[FormScript("Parametros.Tecnicos")]
[BasedOnRow(typeof(TecnicosRow), CheckNames = true)]
public class TecnicosForm
{
    public int IdEmpresa { get; set; }
    public int IdArea { get; set; }
    public string NombreCompleto { get; set; }
    public string Celular { get; set; }
    public string Email { get; set; }
    [TextAreaEditor(Rows = 5)]
    public string Observaciones { get; set; }
    [HideOnInsert]
    public bool Baja { get; set; }
}

File: InPlan.Web/Modules/ServerTypes/Parametros/TecnicosForm.ts
import { LookupEditor, StringEditor, TextAreaEditor, BooleanEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface TecnicosForm {
    IdEmpresa: LookupEditor;
    IdArea: LookupEditor;
    NombreCompleto: StringEditor;
    Celular: StringEditor;
    Email: StringEditor;
    Observaciones: TextAreaEditor;
    Baja: BooleanEditor;
}

export class TecnicosForm extends PrefixedContext {
    static formKey = 'Parametros.Tecnicos';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!TecnicosForm.init)  {
            TecnicosForm.init = true;

            var w0 = LookupEditor;
            var w1 = StringEditor;
            var w2 = TextAreaEditor;
            var w3 = BooleanEditor;

            initFormType(TecnicosForm, [
                'IdEmpresa', w0,
                'IdArea', w0,
                'NombreCompleto', w1,
                'Celular', w1,
                'Email', w1,
                'Observaciones', w2,
                'Baja', w3
            ]);
        }
    }
}